source/connect/tls.txt

```diff
@@ -0,0 +1,267 @@
+.. _php-tls:
+
+========================================
+Configure Transport Layer Security (TLS) 
+========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: security, authentication, transport layer security, encrypt
+
+Overview
+--------
+
+In this guide, you can learn how to use the :wikipedia:`TLS <Transport_Layer_Security>`
+protocol to secure your connection to a MongoDB deployment.
+
+When you enable TLS for a connection, the {+driver-short+} performs the following actions:
+
+- Uses TLS to connect to the MongoDB deployment
+- Verifies the deployment's certificate
+- Ensures that the certificate certifies the deployment
+
+To learn how to configure your MongoDB deployment for TLS, see the
+:manual:`TLS configuration guide </tutorial/configure-ssl/>` in the
+{+mdb-server+} manual. 
+
+.. note::
+
+   This page assumes prior knowledge of TLS/SSL and access to valid certificates.
+   A full description of TLS/SSL, PKI (Public Key Infrastructure) certificates, and
+   Certificate Authorities (CAs) is beyond the scope of this documentation. 
+
+.. tip::
+
+   The {+driver-short+} delegates most TLS behavior to the MongoDB C Driver.
+   For information about how the C driver handles TLS, including configuration steps
+   and expected behavior, see
+   `Configuring TLS <https://www.mongodb.com/docs/languages/c/c-driver/current/libmongoc/guides/configuring_tls/#supported-libraries>`__
+   in the C driver Documentation.
+
+.. _php-enable-tls:
+
+Enable TLS
+----------
+
+To enable TLS for the connection to your MongoDB instance, set the ``tls`` connection
+option to ``true``. You can do this in two ways: by using the ``uriOptions`` parameter
+of the ``MongoDB\Client`` constructor or through a parameter in your connection string.
+
+.. include:: /includes/connect/tls-tabs.rst
```

Comment:
Unsure if this is the desired behavior, but clicking between the tabs here switches the tabs across the whole page. My assumption is because a lot of tabs share the same `:tabid:` of `mongoclient` and `connectionstring`.

If we want all the tabs to be controlled independently, my guess is to change the tabids to be unique.

Reply:
I haven't experimented with the tabids, but you might be right. In any case, this is how all of our pages work afaik, so I think it's okay here